src/astarte-client/client.ts

```diff
@@ -328,6 +332,15 @@ class AstarteClient {
     await this.$post(this.apiConfig.triggers(this.config), toAstarteTriggerDTO(trigger));
   }
 
+  async getTriggerDeliveryPolicyNames(): Promise<string[]> {
+    const response = await this.$get(this.apiConfig.policies(this.config));
+    return response.data;
+  }
+
+  async installTriggerDeliveryPolicy(policy: AstarteTriggerDeliveryPolicyDTO): Promise<void> {
+    await this.$post(this.apiConfig.policies(this.config), policy);
+  }
+
```

Comment:
The code of this file conflicts with the code from `release-1.1` branch.
Please ensure to rebase your branch on top of the updated `release-1.1` branch and resolve the code conflicts